samples/basic/blinky/README.rst

```diff
@@ -14,13 +14,13 @@ Requirements
 ************
 
 The demo assumes that an LED is connected to one of GPIO lines. The
-sample code is configured to work on boards with user defined buttons and that
-have defined the LED0\_* variables in :file:`board.h`.
+sample code is configured to work on boards with that have defined the led0
+alias in board device tree description file.
 
-The :file:`board.h` must define the following variables:
+Doing so, following variables will be generated:
```

Comment:
delete blank line 19 to put the sentence on line 20 in the same paragraph as the previous sentence.